renderer/components/audio-player.tsx

```diff
@@ -0,0 +1,6 @@
+import dynamic from 'next/dynamic';
+
+const AudioPlayer =
+    typeof window !== 'undefined' ? dynamic(() => import('./audio-player-base'), { ssr: false }) : () => null;
```

Comment:
How come this is needed? Does line 40 of `audio-player-base` not handle the ssr case?

Reply:
@julesyoungberg Not quite since merley importing wavesurver.js uses the window in some way.  This is likely overkill but I found I needed all this for hot module replacement to work.

Reply:
ahh I see, sounds good :+1: